.github/workflows/ost.yaml

```diff
@@ -0,0 +1,19 @@
+name: OST trigger
+
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+    inputs: 
+
+jobs:
+  trigger-ost:
+    if: |
+      ${{ github.event.issue.pull_request &&
+          startsWith(github.event.comment.body, '/ost') &&
+          ( github.event.comment.author_association == 'MEMBER' ||
+            github.event.comment.author_association == 'COLLABORATOR' )
+      }}
+    uses: oVirt/ovirt-system-tests/.github/workflows/ost.yaml@master
+    with:
+      pr_url: ${{ github.event.issue.pull_request.url }} }} ) }}
```

Comment:
please fix parenthesis

Reply:
ack - done!  This was a result some sort of copy-paste black magic. Thanks for spotting it.